config/routes.rb

```diff
@@ -1011,6 +1011,7 @@
         pod_metrics_data
         project_data
         show
+        refresh_status_data
```

Comment:
I don't think this should be here, as we don't want to expose refresh_status_data as a route.  I think it should just be a private method on the controller.

Reply:
@Fryguy here last_refersh_status is end point, that is why added in routes file.

Reply:
:+1:

Reply:
Can you move this up alphabetically?

Reply:
done.